testcases/OpTestEEH.py

```diff
@@ -306,18 +306,24 @@ def get_dic_of_pe_vs_addr(self):
     #
     def run_pe_4(self, addr, e, f, phb, pe, con):
         self.prepare_logs()
-        count_old = self.check_eeh_slot_resets()
+        slot_reset_count_old = self.check_eeh_slot_resets()
+        interface_count_old = self.get_count_of_interdaces_in_pci_devices(pe)
         rc = self.inject_error(addr, e, f, phb, pe)
         if rc != 0:
             print "Skipping verification as command failed"
             return
         # Give some time to EEH PCI Error recovery
         time.sleep(60)
-        count = self.check_eeh_slot_resets()
-        if int(count) > int(count_old):
+        slot_reset_count = self.check_eeh_slot_resets()
+        if int(slot_reset_count) > int(slot_reset_count_old):
             print "PE Slot reset happenned successfully on pe: %s" % pe
         else:
             print "PE Slot reset not happened on pe: %s" % pe
+        interface_count = self.get_count_of_interdaces_in_pci_devices(pe)
+        if interface_count > interface_count_old:
```

Comment:
How interface_count will increase here after error injection?
it should be 
if interface_count == interface_count_old:
      print "PE interfaces recovery happenned successfully on pe: %s" % pe
else:
      print "PE interfaces recovery not happened on pe: %s" % pe